src/library-authoring/import-course/MigratedCourseCard.test.tsx

```diff
@@ -0,0 +1,94 @@
+import userEvent from '@testing-library/user-event';
+
+import {
+  initializeMocks,
+  render as testRender,
+  screen,
+  waitFor,
+} from '@src/testUtils';
+
+import { LibraryProvider } from '../common/context/LibraryContext';
+import {
+  mockContentLibrary,
+  mockGetCourseMigrations,
+} from '../data/api.mocks';
+import { type CourseMigration } from '../data/api';
+import { MigratedCourseCard } from './MigratedCourseCard';
+
+initializeMocks();
```

Comment:
We need to use `initializeMocks` inside the `beforeEach`, because it calls `clearAllMocks` to clear the history in every test